src/settings/__tests__/settingsSelectors-test.js

```diff
@@ -0,0 +1,69 @@
+/* @flow */
```

Comment:
Tests are good.
These though might be a bit too straightforward and almost impossible to break.
Maybe we don't need them.